pkg/api/vlabs/validate.go

```diff
@@ -666,6 +678,13 @@ func validateUniquePorts(ports []int, name string) error {
 	return nil
 }
 
+func validateKubernetesLabelValue(v string) error {
+	if !(len(v) == 0) && !labelValueRegex.MatchString(v) {
```

Comment:
dropped the length check

Reply:
Yep I did that on purpose. I added it to the regexp instead, that way i's cleaner. I forgot to delete the length const, I'll push that with my next commit.